module/zstd/README.md

```diff
@@ -22,36 +21,19 @@ Tree structure:
 To update ZSTD the following steps need to be taken:
 
 1. Grab the latest release of [ZSTD](https://github.com/facebook/zstd/releases).
-2. Update `module/zstd/zstd-in.c` if required. (see
-   `zstd/contrib/single_file_libs/zstd-in.c` in the zstd repository)
-3. Generate the "single-file-library" and put it to `module/zstd/lib/`.
-4. Copy the following files to `module/zstd/lib/`:
-   - `zstd/lib/zstd.h`
-   - `zstd/lib/common/zstd_errors.h`
+2. Copy the files output by the following script to `module/zstd/lib/`:
+`grep include [path to zstd]/contrib/single_file_libs/zstd-in.c  | awk '{ print $2 }'`
+3. Remove debug.c, threading.c, and zstdmt_compress.c.
```

Comment:
Since I'm being pedantic: other filenames are Courierised (L14), so it may make sense to Courierise these too.